Formula/csound.rb

```diff
@@ -29,18 +33,39 @@ def install
       -DBUILD_FLUID_OPCODES=OFF
       -DBUILD_JAVA_INTERFACE=OFF
       -DBUILD_LUA_INTERFACE=OFF
-      -DBUILD_PYTHON_INTERFACE=OFF
       -DCMAKE_INSTALL_RPATH=#{frameworks}
     ]
 
     mkdir "build" do
       system "cmake", "..", *args
       system "make", "install"
 
-      include.install_symlink "#{frameworks}/CsoundLib64.framework/Headers" => "csound"
+      macho = MachO.open("_csnd6.so")
+      macho.change_install_name("@rpath/CsoundLib64.framework/Versions/6.0/CsoundLib64", "#{frameworks}/CsoundLib64.framework/CsoundLib64")
+      macho.change_install_name("@rpath/libcsnd6.6.0.dylib", "#{lib}/libcsnd6.dylib")
+      macho.delete_rpath(pwd.sub(%r{^/private/}, "/"))
```

Comment:
Shouldn't upstream be responsible for providing a sane build system with proper install names? I don't think it make sense for us to be doing this much tweaking of the installation.

Reply:
I’d say yes. If this is a deal-breaker here, I can open a pull request upstream.

Reply:
Yup, I think this is a deal breaker for us. A pr upstream with a patch here (once they show interest in merging it) is far preferable.

Thanks!

Reply:
OK, thanks for the review.

Reply:
On second thought, one of the two Python bindings doesn’t require changing the installation like this, so this is now removed, but this appears to be one-step-forward-and-two-steps back—the test failure is because of the recently added audit for linking against Accelerate (https://github.com/Homebrew/brew/commit/35a271e99b656d49c0fd44356a68558143942aed).